docs/api/index.md

```diff
@@ -1886,7 +1886,7 @@ function shallowMount(Component, options?: MountingOptions): VueWrapper
 **Signature:**
 
 ```ts
-enableAutoUnmount(hook: Function));
+enableAutoUnmount(hook: (callback: Function) => void));
```

Comment:
Maybe even `hook: (callback: () => void) => void`?

Reply:
I will update types, thx.

Reply:
To be coherent, maybe change the type here in the docs as well to be the same as the TS types?